### PR TITLE
feat: Add gaze.from_ipc()

### PR DESCRIPTION
**Pull request by prassepaul**
## Description

add functionality to read from IPC/feather

Fixes issue #518

## Implemented changes

Insert a description of the changes implemented in the pull request.

- [x] added support for IPC/feather

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by SiQube — APPROVED** [review]

File: src/pymovements/gaze/io.py [hunk 1]
```diff
@@ -158,3 +158,59 @@ def from_csv(
         acceleration_columns=acceleration_columns,
     )
     return gaze_df
+
+
+def from_ipc(
+        file: str | Path,
+        experiment: Experiment | None = None,
+        **read_csv_kwargs: Any,
```
Comment (dkrako) [inline src/pymovements/gaze/io.py:166]: rename to read_ipc_kwargs

File: src/pymovements/gaze/io.py [hunk 2]
```diff
@@ -158,3 +158,59 @@ def from_csv(
...
+) -> GazeDataFrame:
+    """Initialize a :py:class:`pymovements.gaze.gaze_dataframe.GazeDataFrame`.
+
+    Parameters
+    ----------
+    file:
+        Path of IPC/feather file.
+    experiment : Experiment
+        The experiment definition.
+    **read_csv_kwargs:
```
Comment (dkrako) [inline src/pymovements/gaze/io.py:176]: update to read_ipc_kwargs

File: src/pymovements/gaze/io.py [hunk 3]
```diff
@@ -158,3 +158,59 @@ def from_csv(
...
+            Additional keyword arguments to be passed to polars to read in the csv.
+
+    Examples
+    --------
+    First let's assume a IPC file stored `tests/gaze/io/files/monocular_example.feather`
+
+    We can now load the data into a ``GazeDataFrame`` by specyfing the experimental setting
```
Comment (dkrako) [inline src/pymovements/gaze/io.py:183]: this sentence needs to be update, the experimental setting is not specified in this example

File: src/pymovements/datasets/__init__.py [hunk 4]
```diff
@@ -53,6 +54,7 @@
     'GazeBaseVR',
     'GazeOnFaces',
     'JuDo1000',
+    'GazeOnFaces',
```
Comment (dkrako) [inline src/pymovements/datasets/__init__.py:57]: something went wrong here I think

**Review by dkrako — CHANGES_REQUESTED** [review]
Thanks a lot for your work!

As always, I have to nitpick about some minor details.

File: src/pymovements/gaze/io.py (hunk 1 above)
Comment (prassepaul) [reply src/pymovements/gaze/io.py:166]: haha. fair point

File: src/pymovements/gaze/io.py (hunk 2 above)
Comment (prassepaul) [reply src/pymovements/gaze/io.py:176]: k

File: src/pymovements/gaze/io.py (hunk 3 above)
Comment (prassepaul) [reply src/pymovements/gaze/io.py:183]: k

File: src/pymovements/datasets/__init__.py (hunk 4 above)
Comment (prassepaul) [reply src/pymovements/datasets/__init__.py]: fixed

**Review by dkrako — APPROVED** [review]

**Pull request merged**